### PR TITLE
Return host IP when using vmware as vm driver.

### PR DESCRIPTION
**Pull request by ne0h**
Adds the missing case "vmware" to return the host IP.

### CODE REVIEW

**Comment by ne0h** [comment]
Signed CLA.

**Comment by tstromberg** [comment]
Hey ne0h!

I'm a bit confused by this PR: the title says it implements mounting, but the PR adds an IP. 

Can you elaborate some in the PR description as to how the two are related? Thanks!

**Comment by tstromberg** [comment]
@minikube-bot OK to test

**Comment by ne0h** [comment]
Hey @tstromberg, you are absolutely right. I updated the commit and the description of the PR to be more applicable.

**Comment by ne0h** [comment]
Given that `host.Driver.GetIP()` returns the guest IP address reliably, the switch statement could be removed completely?

**Comment by tstromberg** [comment]
> Given that `host.Driver.GetIP()` returns the guest IP address reliably, the switch statement could be removed completely?

I wonder about this. I'd be happy to merge this PR as is, but if this simplification works, that's great. I'm concerned however that some drivers may have different values for host.Driver.GetIP() and the VM-Host IP, but the integration tests should catch that. Feel free to propose in a different PR if you like.

File: pkg/minikube/cluster/cluster.go [hunk 1]
```diff
@@ -380,6 +380,16 @@ func GetVMHostIP(host *host.Host) (net.IP, error) {
 		return ip, nil
 	case "xhyve", "hyperkit":
 		return net.ParseIP("192.168.64.1"), nil
+	case "vmware":
+		vmIPString, err := host.Driver.GetIP()
+		if err != nil {
+			return []byte{}, errors.Wrap(err, "Error getting VM IP address")
+		}
+		vmIP := net.ParseIP(vmIPString).To4()
```
Comment (tstromberg) [inline pkg/minikube/cluster/cluster.go]: To be consistent with the others, I think you can just call:

return net.ParseIP(vmIPstring), nil

**Pull request merged**